com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -485,10 +628,38 @@ private void SendAllParamsAndState()
             }
         }
 
+        private ulong[] GetTargetClientIds(ulong originClientId)
+        {
+
+            if (m_TargetClientIdsCache == null)
+            {
+                m_TargetClientIdsCache = new Dictionary<ulong, ulong[]>();
+            }
+
+            if (!m_TargetClientIdsCache.TryGetValue(originClientId, out var ids))
```

Comment:
I would wrap this in an else block to make clearer; these 2 if's are mutually exclusive